client/src/driver/BitBakeProjectScanner.ts

```diff
@@ -343,8 +343,21 @@ You should adjust your docker volumes to use the same URIs as those present on y
 
       const extraInfo = [`layer: ${layerName}`, `version: ${version} `].join('\r\n')
 
+      /**
+       * The output of 'bitbake-layers show-layers' is like this:
+       * layer                 path                                                              priority
+         =================================================================================================
+         core                  /home/projects/poky/meta                                          5
+
+         The output of 'bitbake-layers show-recipes' is like this:
+         acl:
+         meta                 2.3.2
+
+         Here 'meta' is used to refer to the layer instead of 'core'. So in such case we need to compare
+         the basename of the path found in 'show-layers' with the layer name found in this function.
+       */
       const layerInfo = this._bitbakeScanResult._layers.find((layer) => {
-        return layer.name === layerName
+        return layer.name === layerName || path.parse(layer.path).name === layerName
```

Comment:
Interestingly, if a project scan has been cached from the previous version, it will still contain the previous full paths which are "buggy". Honestly I'm not sure if we should invalidate the previous Project scan caches just for this.

Reply:
We could mention this in the changelog but it is definitely better that we do this for the users to improve their experience. 
My idea is to invalidate the cache results every time upon installing/updating the extension. The cached results are stored in the `workspaceState`, and they persist between extension installation/uninstallation. We could remove them upon uninstallation.

Reply:
Don't hesitate to implement the cache invalidation logic in another PR!
Are you sure uninstalling is triggered when updating?
While we are implementing this, we could do it the way it should: All serialization libraries allow to add a data format version. We should do that. When saving/loading the cache, prefix the data with a version field. Discard it if the version is not the same as the most recent one.

Reply:
From my research, there is no hook that we can use to do something upon extension installation. Keeping track of the format version is a good idea. A simple integer should do.

Reply:
We will also need several tests and fixtures for it

Reply:
Will do it in another PR.